CMakeLists.txt

```diff
@@ -250,7 +256,14 @@ else ()
     endif ()
 
     add_subdirectory("include")
-    add_subdirectory("tests")
-    add_subdirectory("tools")
-    add_subdirectory("examples")
+
+    if (CPPKAFKA_ENABLE_TESTS)
```

Comment:
A simple flag (e.g. `TO_BUILD_ADDTIONAL_BINARIES`) should be OK.
How about the following?
```
if (TO_BUILD_ADDTIONAL_BINARIES)
    add_subdirectory("tests")
    add_subdirectory("tools")
    add_subdirectory("examples")
endif()
```

Reply:
I've maid separate variable to expose it as a CACHE variable so that the user of `ExternalProject` could configure it. But this is not something I care about much :)